tasks/chkrootkit.yml

```diff
@@ -1,5 +1,13 @@
+- name: Generate list of enabled chkrootkit tests
+  shell: chkrootkit -l 2>&1 | cut -d ' ' -f 3- | sed -r "s/\b({{CHKROOTKIT_DISABLED_TESTS | join('|')}})\b//g"
```

Comment:
I was actually thinking of embedding this code in the configuration file itself, but your solution is even better.  This way, the configuration file is still just a configuration file and not a script.

I think you should be using single quotes in the `sed` command, just for safety.  (I don't think `\b` is translated in any way in double quoted strings, but I tend to use single quotes whenever I can.)